consul/resource_consul_service_splitter_config_entry.go

```diff
@@ -0,0 +1,255 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package consul
+
+import (
+	"fmt"
+	"strings"
+
+	consulapi "github.com/hashicorp/consul/api"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+const KindServiceSplitter = "service-splitter"
+
+var serviceSplitterConfigEntrySchema = map[string]*schema.Schema{
+	"name": {
+		Type:     schema.TypeString,
+		Required: true,
+		ForceNew: true,
+	},
+	"kind": {
```

Comment:
I think we don't need the kind here because it should always be service-splitter

Reply:
This is computed. So user will not be able to enter this.

Reply:
This field is computed. There is not interaction of this field with user. Also I am having trouble removing it because 
```				sw.set("kind", kind)
				sw.set("name", name)
				sw.set("partition", partition)
				sw.set("namespace", namespace)
```
state writer is setting in resouces data itself.

Reply:
What is the issue in keeping it this way?